home_work_23/ViewController.swift

```diff
@@ -11,7 +11,31 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        // адрес сервера
+        if let url = URL(string: "https://api.openweathermap.org/data/2.5/weather?q=London&appid=3b0d514cd0fcc25c0056ef824e4464ad") {
```

Comment:
Please move out of url string queries values for params `q` and `appid`
ApiKey should be stored correctly.